variants/heltec_wireless_stick_lite_v3/pins_arduino.h

```diff
@@ -0,0 +1,60 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define Wireless_Stick_Lite true
+#define DISPLAY_HEIGHT 0
+#define DISPLAY_WIDTH  0
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       15
```

Comment:
`NUM_ANALOG_INPUTS` should be 20 for ESP32-S3

Reply:
Fixed